odk1-src/central-projects.rst

```diff
@@ -31,16 +31,19 @@ If you have upgraded to **version 0.5**, the following changes have been made:
    - Create, manage, and retire App Users for the project.
    - Manage and archive the project itself.
 
-In future releases beyond 0.5, we have a `loose roadmap <https://github.com/opendatakit/central/issues/35>`_ with at least the following goals:
+Since the major changes that occurred in version 0.5, we have additionally made the following improvements:
+
+ - Better, centralized form workflow/status management.
+ - More granular project access, in the form a Project Viewer role which allows read-only access only to Forms and Submission data.
```

Comment:
s/the form a/the form of a/

Reply:
Optional: To match the style of the previous text, you could change the first sentence to "In <bold>version 0.6</bold>, we have additionally made the following improvements"

Reply:
no, i don't want to turn this into a huge messy by-version changelog. big things happened in 0.5, and we've made incremental changes since then.